.github/workflows/pythonapp.yml

```diff
@@ -35,10 +35,10 @@ jobs:
       if: ${{ matrix.python-version == '3.5' && matrix.os != 'macos-latest' }}
       run: |
         pip install torch==1.5.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
-    - name: Install pytorch 1.6.0+cpu if python > 3.5 (not mac)
+    - name: Install pytorch 1.7.0+cpu if python > 3.5 (not mac)
```

Comment:
do we need to also pin line 33 to a specific version?

Reply:
yes! Good catch!